modules/caddyhttp/reverseproxy/caddyfile.go

```diff
@@ -803,6 +803,7 @@ func (h *Handler) FinalizeUnmarshalCaddyfile(helper httpcaddyfile.Helper) error
 //         read_buffer             <size>
 //         write_buffer            <size>
 //         max_response_header     <size>
+//         proxy_protocol_version  [off|v1|v2]
```

Comment:
We tend to use `[]` to mean "optional", so this shouldn't have those

```suggestion
//         proxy_protocol_version  off|v1|v2
```

Reply:
What's the purpose of `off` though? I don't understand why we would offer that option. Not configuring `proxy_protocol_version` means off.

Reply:
No particular reason, just like strict_sni_host global option which has an "off".

Reply:
I think we should rename this to `proxy_protocol` and I agree with removing the `off` option, since that is the same as omitting it entirely.